.github/workflows/release.yml

```
@@ -0,0 +1,18 @@
+name: Create GitHub Release
+
+on:
+  # Run on demand
+  workflow_dispatch:
+
+  # Run on push to main when .bumpversion.cfg version is updated
+  push:
+    branches:
+      - main
+    paths:
+      - .bumpversion.cfg
+
+jobs:
+  release:
+    uses: plus3it/actions-workflows/.github/workflows/release.yml@41056300ba78d8be98b45f057aea246f8c4f30cf
+    secrets:
```

In the `.travis-ci.yaml` file, we can see this project only invoked two jobs, for lint and deploy. That was because the test job that runs against the mockstack is using apis that are not implemented by localstack or moto server. In these cases, we need to exclude the test job from the release workflow:

```suggestion
    uses: plus3it/actions-workflows/.github/workflows/release.yml@41056300ba78d8be98b45f057aea246f8c4f30cf
    with:
      mockstacktest-enable: false
    secrets:
```